### ISSUE TITLE
Use :dir instead of attribute selector

### ISSUE THREAD

**Issue by esprehn** [PolymerElements/paper-checkbox#107]
`:host-context([dir="rtl"]) #ink`

should be

#ink:dir(rtl) I think?

:host-context and attribute selectors will be expensive.

**Comment by bicknellr** [comment]
I agree, but I don't think `:dir` is supported anywhere except Firefox yet (and prefixed there still). [I added a similar hack to paper-tabs.](https://github.com/PolymerElements/paper-tabs/blob/f0abeab2a174770cf4119272e2a87945b254afa7/paper-tabs.html#L113)

**Comment by DavidHenri008** [comment]
@bicknellr the usage of :host-context([dir="rtl"]) is also done in the paper-checkbox and the result is awful on edge...
